examples/git-credentials/git-ssh-auth-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
Seems like this is a intermedium generated file? If so, it shouldn't be commited.